Gopkg.toml

```diff
@@ -9,7 +9,8 @@ required = [
 
 [[override]]
   name = "github.com/gardener/gardener"
-  revision = "8725569dc996defe240950cc71cc4509a29c91fe"
+  source = "github.com/swapnilgm/gardener"
```

Comment:
Once https://github.com/gardener/gardener/pull/1079 is merged you should file another PR for the https://github.com/gardener/gardener/tree/gardener-extensions-fixes branch

Reply:
Yeah Sure.